puppet/manifests/development.pp

```diff
@@ -5,8 +5,6 @@
 $extensions = sz_extensions('/vagrant/extensions')
 $php_extensions = [ 'curl', 'gd', 'mysql' ]
 
-Class['mysql'] -> Package['php5-mysql']
-
```

Comment:
I think we still need this dependency; should it be moved into php.pp as `${php_package}-mysql`?